tests/conftest.py

```diff
@@ -40,6 +39,14 @@ def pytest_configure():
         PASSWORD_HASHERS=(
             'django.contrib.auth.hashers.MD5PasswordHasher',
         ),
+        **{
```

Comment:
2. I could have kept it in the same location, but it would have seemed weird to have a kwarg expansion in the middle of the argument assignments.